docs/source/running_mypy.rst

```diff
@@ -382,26 +382,29 @@ the library, you will get a message like this:
     main.py:1: note: (or run "mypy --install-types" to install all missing stub packages)
 
 You can resolve the issue by running the suggested pip commands.
+
 If you're running mypy in CI, you can ensure the presence of any stub packages
 you need the same as you would any other test dependency, e.g. by adding them to
-the appropriate ``requirements.txt`` file.
+the appropriate ``requirements.txt`` file or dependency section of ``pyproject.toml``.
 
-Alternatively, add the :option:`--install-types <mypy --install-types>`
-to your mypy command to install all known missing stubs:
+The :option:`--install-types <mypy --install-types>` flag
+makes mypy list and (after a prompt) install all known missing stubs:
 
 .. code-block:: text
 
     mypy --install-types
 
 This is slower than explicitly installing stubs, since it effectively
-runs mypy twice -- the first time to find the missing stubs, and
+runs mypy twice — the first time to find the missing stubs, and
```

Comment:
Why use the literal dash here? I quite love the ability to edit files without having to type AltGR-hyphen-hyphen and hoping I can remember the compose key combo correctly (spoiler: I don't, it's actually AltGR-hyphen-hyphen-hyphen)... Sphinx already converts literal `--` to en-dash, so why make it more difficult to type?